app/lib/features/users/widgets/user_info_drawer.dart

```diff
@@ -0,0 +1,141 @@
+import 'package:acter/common/providers/common_providers.dart';
+import 'package:acter/features/member/widgets/member_info_skeleton.dart';
+import 'package:acter/features/users/widgets/message_user_button.dart';
+import 'package:acter/features/users/providers/user_profile_provider.dart';
+import 'package:acter_avatar/acter_avatar.dart';
+import 'package:acter_flutter_sdk/acter_flutter_sdk_ffi.dart';
+import 'package:flutter/material.dart';
+import 'package:flutter/services.dart';
+import 'package:flutter_easyloading/flutter_easyloading.dart';
+import 'package:flutter_gen/gen_l10n/l10n.dart';
+import 'package:flutter_riverpod/flutter_riverpod.dart';
+import 'package:logging/logging.dart';
+import 'package:acter/common/extensions/options.dart';
+
+final _log = Logger('a3::user::user_info_drawer');
+
+class _UserInfoDrawerInner extends ConsumerWidget {
```

Comment:
mostly a copy of the member_info_drawer. Just simplified and removed a few things we never have here.